### PR TITLE
feat(container): add settings

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

[> Link to the JIRA ticket
](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-525)

- Add settings for Containers

Capture d’écran 2022-09-13 à 16 54 53

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [ x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.  
Click on the link below to follow its deployment and use it.  
:point_right: [[PR] staging - feat(container): add settings - 2022-09-13T14:55:23Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/60517f53-f2cf-4b11-ae7a-1c83ca90d63c/applications)

File: libs/pages/application/src/lib/feature/page-settings-general-feature/page-settings-general-feature.tsx [hunk 1]
```diff
@@ -14,41 +15,43 @@ export const buildGitRepoUrl = (provider: string, branch: string): string => {
...
 }
 
-export const handleSubmit = (data: FieldValues, application: GitApplicationEntity) => {
-  const cloneApplication = Object.assign({}, application as GitApplicationEntity)
+export const handleSubmit = (data: FieldValues, application: ApplicationEntity) => {
+  const cloneApplication = Object.assign({}, application)
   cloneApplication.name = data['name']
 
-  cloneApplication.build_mode = data['build_mode']
+  if ('build_mode' in cloneApplication) {
```
Comment (bdebon) [inline libs/pages/application/src/lib/feature/page-settings-general-feature/page-settings-general-feature.tsx:22]: eheh nice!

File: libs/pages/application/src/lib/page-application.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ export function PageApplication() {
     equal
   )
 
-  useDocumentTitle(`${application?.name} - Qovery`)
+  useDocumentTitle(`${application?.name || 'Application'} - Qovery`)
```
Comment (bdebon) [inline libs/pages/application/src/lib/page-application.tsx:40]: Nice!

**Review by bdebon — APPROVED** [review]
Did not test but the code is nickel!

**Pull request merged**